src/app/features/event/singleEvent.component.html

```diff
@@ -0,0 +1,88 @@
+<ng-container *ngIf="eventInfo$ | async as eventInfo">
+  <div class="event">
+    <ng-container *ngIf="eventInfo.group">
+      <div class="event__group-private-label">
+        <div class="event__group-private-label__text">
+          <p class="font-text-1">
+            Grupa
+            <br />
+            {{ eventInfo.group.name | titlecase }}
+          </p>
+        </div>
+      </div>
+    </ng-container>
+    <ng-container *ngIf="eventInfo.isPrivate"
+      ><div class="event__group-private-label">
+        <div class="event__group-private-label__text">
+          <div class="event__group-private-label__text__icons">
+            <p class="font-text-1">
+              Wydarzenie <br />
+              prywatne
+            </p>
+            <mat-icon fontIcon="lock" class="event__white-icon"></mat-icon>
+          </div>
+        </div>
+      </div>
+    </ng-container>
+    <h1 class="$heading-1">{{ eventInfo.name | titlecase }}</h1>
+    <ng-container *ngIf="eventInfo.isConfirmationRequired">
+      <div class="event__required-confirmation">
+        <p class="font-text-1" class="event__required-confirmation__text">ZAPIS OBOWIĄZKOWY</p>
```

Comment:
1. BEM 
2. jak chcemy używac WIELKICH liter to używajmy do tego SCSS'a property: `text-transform: uppercase;`